commands/management/add_mysql.go

```diff
@@ -268,5 +274,6 @@ func init() {
 	AddMySQLC.Flag("skip-connection-check", "Skip connection check").BoolVar(&AddMySQL.SkipConnectionCheck)
 	AddMySQLC.Flag("tls", "Use TLS to connect to the database").BoolVar(&AddMySQL.TLS)
 	AddMySQLC.Flag("tls-skip-verify", "Skip TLS certificates validation").BoolVar(&AddMySQL.TLSSkipVerify)
+	AddMySQLC.Flag("create-user", "Create user root").Hidden().BoolVar(&AddMySQL.CreateUser)
```

Comment:
This is minor but I guess it's better to be
```suggestion
	AddMySQLC.Flag("create-user", "Create pmm user").Hidden().BoolVar(&AddMySQL.CreateUser)
```
according to https://www.percona.com/doc/percona-monitoring-and-management/conf-mysql.html#pmm-conf-mysql-user-account-creating